contents/docs/_snippets/capture-group-event-code.mdx

```diff
@@ -1,13 +1,16 @@
 <MultiLanguage>
 
 ```js-web
-// Option 1 (recommended): Call posthog.group()
-// This has the side-effect that all subsequent events in the session are associated to the group
+// Call posthog.group() to create a group before capturing events.
+// It sends a `$groupidentify` event to create or update the group.
+// It will also create the group type if it doesn't exist. In the
+// web SDK, it also associates all subsequent events in the session
+// with the group.
 posthog.group('company', 'company_id_in_your_db');
 posthog.capture('user_signed_up'); // This event is associated with the company above
 
-// Option 2: Set the group property in posthog.capture()
-// This method doesn't have the side-effect of associating the session's events to the group
+// If the group type is already created, you can simply set the
+// `$groups` property in `capture()` for subsequent sessions.
```

Comment:
This sort of implies, you should prefer this in subsequent sessions where the group already exists? Is that correct? Or should you still call `group()`?

Reply:
> This sort of implies, you should prefer this in subsequent sessions where the group already exists? Is that correct? Or should you still call `group()`?

Hm, I guess I don't actually have a strong opinion. I think including `$groups` in the `capture()` call makes it easier to verify expected behavior, but I'm not sure what real-world patterns are. Is there a different phrasing you'd suggest?

Reply:
Maybe?
```suggestion
// If the group type is already created, you can also manually add the 
// `$groups` property to any events you want to associate with the group
```

Reply:
Yeah, that's better. Thanks! 385c43d2b6928be2a43434cdddb4e5d3e3238937